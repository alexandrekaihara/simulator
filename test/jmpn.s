LOAD MINUSONE
JMPN ISLESSZERO
OUTPUT MINUSONE
STOP
ISLESSZERO:
LOAD ONE
JMPN ISGREATERZERO
OUTPUT ONE
STOP
ISGREATERZERO:
OUTPUT MINUSONE
STOP
ONE: CONST 1
MINUSONE: CONST -1
