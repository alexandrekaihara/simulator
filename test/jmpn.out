10 18 6 7 13 18 14 10 17 6 14 13 17 14 13 18 14 1 -1
